NEWS.md

```diff
@@ -1,5 +1,11 @@
 # connectapi (development version)
 
+## New features
+
+- `get_users()` now supports filtering users with the `account_status` and
+  `user_role` parameters. This allows you to, for example, find all licensed
+  users on a Connect server. (#311)
```

Comment:
```suggestion
- `get_users()` now supports filtering users with the `account_status` and
  `user_role` parameters. For example, this allows you to find all licensed
  users on a Connect server. (#311)
```

Reply:
Super minor tweak, but trying to make the example bit there to not break up the rest of the sentence